### ISSUE TITLE
Places API lookup bug

### ISSUE THREAD

**Issue by DevinWalker** [WordImpress/google-maps-builder#6]
Qhen you query a ton of place types in a radius it results in a JS error and quits working leaving the spinner in place. 

Screenshot of console error:

![google-maps-bug](https://cloud.githubusercontent.com/assets/1571635/8139531/535e530e-1109-11e5-9ea3-3302af246ef3.png)

**Comment by DevinWalker** [comment]
Present in both the front and backend: "Failed to load resource: the server responded with a status of 400 (Bad Request)"

**Comment by royboy789** [comment]
Where in the admin is this? Is this under the map when creating a new map "Google Places" meta box with the checkboxes?

**Comment by DevinWalker** [comment]
Yeah, exactly. Create a new map. Select ALL places to display within a radius of 3,000 meters. You should see the map start creating place markers then fail with the JS error posted above in console.

**Comment by royboy789** [comment]
@DevinWalker - I"m not sure but have you checked the API restriction on calls / minute, I know the API has some restrictions like that, not sure if this is the cause, in which case we'd have to find a way to either slow it down or limit results before it keeps on requesting.

**Comment by DevinWalker** [comment]
I think I may have done something like that awhile back... I know I did that for geocoding and maybe places too

**Comment by DevinWalker** [comment]
I have a thought that this error occurs when a lookup happens with corrupt data. I haven't been able to reproduce on maps with a small number of markers

**Comment by DevinWalker** [comment]
I'm removing from 2.0 milestone unless it comes up in beta testing.

**Comment by nishitlangaliya** [comment]
Slack Call Summary

Participants: @nishitlangaliya, @ravinderk
Topic: As discuss on call I need google API credentials for further development. As per google new policies https://developers.google.com/places/web-service/usage-and-billing?hl=en_US requires billing details needs to add. 
Result: I will send email to @DevinWalker regarding requesting api credentials for the same.

**Comment by nishitlangaliya** [comment]
@DevinWalker I need google map api for further development. Would you please check your email ?

Thanks

**Comment by DevinWalker** [comment]
@nishitlangaliya you've been provided the API key. Moving back to you Sprint Backlog.

**Comment by nishitlangaliya** [comment]
@DevinWalker ,

I have tried to reproduce the case but unable to reproduce error on my local server.

**Comment by nishitlangaliya** [comment]
@DevinWalker ,

I have tried to reproduce error but not able to reproduce case. Below cases I test to reproduce.

1. Add Google map in admin side with google places. 
2. Selected all places and create map.
3. Test with 300, 3000 , 50000 radius with high speed and also with 2G speed. 

So is it any specific location or lat long using which we can reproduce case.

Thanks
